src/opening-period/v2/components/form/OpeningHoursForm.tsx

```diff
@@ -200,81 +204,84 @@ const OpeningHoursForm = ({
     (resource && datePeriodConfig && (
       <FormProvider {...form}>
         <form onSubmit={form.handleSubmit(onSubmit)}>
-          <div className="opening-hours-page">
-            <ResourceTitle
-              openingHours={openingHours}
-              resource={resource}
-              resourceStates={resourceStates}
-              rules={rules}
-            />
-            <Accordion card heading="Ohjeet">
```

Comment:
Wrapped this area with a section so that could use the minus margin only for that section to get the resource header to behave nicely for both resource page and this page.